pingdom/provider.go

```diff
@@ -11,24 +11,27 @@ import (
 func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"user": &schema.Schema{
+			"user": {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
 
-			"password": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
+			"password": {
+				Type:      schema.TypeString,
+				Optional:  true,
+				Sensitive: true,
 			},
 
-			"api_key": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
+			"api_key": {
+				Type:      schema.TypeString,
+				Optional:  true,
+				Sensitive: true,
 			},
-			"account_email": &schema.Schema{
-				Type:     schema.TypeString,
-				Default:  "",
-				Optional: true,
+			"account_email": {
```

Comment:
In my opinion `account_email` could be left out as not sensitive. Everything else it's perfect!

Reply:
Yeah I was on the fence about that one.  I was thinking that leaking someone's email in log files is probably not ideal but also not critical either.